app/components/forms/bill-of-lading/template.hbs

```diff
@@ -18,10 +18,14 @@
 <div class="rowContainer">
   {{#each validOrderitems as |orderItem|}}
     {{rows/bill-of-lading-row
-        model=orderItem}}
+        name=orderItem.item.name
+        unitPrice=orderItem.unitPrice
+        quantity=orderItem.quantity
+        total=orderItem.total}}
```

Comment:
Let's remove this prop and instead compute it based on the unitPrice and quantity. This will avoid any calculation errors that may be introduced by manually setting it. Also removes one more dependency.